ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+Welcome to the issues section if it's your first time!
+
+### Before creating an issue, please ensure:
+- [ ] This is a bug in the software that resides in this repository, and not a
+      support matter (use https://otland.net/forums/support.16/ for support)
+- [ ] You have not made changes to the source code related to the problem
+- [ ] Search for a similar issue, add your reaction there instead :tada:
```

Comment:
Better not to have or to rewrite to `add your reaction and complement information`?